modules/nodes-nodes-audit-log-basic.adoc

```diff
@@ -3,30 +3,34 @@
 // * nodes/nodes-audit-log.adoc
 
 [id="nodes-pods-audit-log-basic_{context}"]
-= About the API Audit Log
-
-Audit works at the API server level, logging all requests coming to the server.
-Each audit log contains two entries:
-
-. The request line containing:
-.. A Unique ID allowing to match the response line (see #2)
-.. The source IP of the request
-.. The HTTP method being invoked
-.. The original user invoking the operation
-.. The impersonated user for the operation (`self` meaning himself)
-.. The impersonated group for the operation (`lookup` meaning user's group)
-.. The namespace of the request or <none>
-.. The URI as requested
-
-. The response line containing:
-.. The unique ID from #1
-.. The response code
-
-You can view logs for the master nodes for the {product-title} API server or the Kubernetes API server.
-
-Example output for the Kubelet API server:
+= About the API audit log
+
+Audit works at the API server level, logging all requests coming to the server. Each audit log contains the following information:
+
+.Audit log fields
+[cols="1,2",options="header"]
+|===
+|Field |Description
+|`level` | The audit level at which the event was generated.
+|`auditID` |A unique audit ID, generated for each request.
```

Comment:
A unique -> An unique

Reply:
"A unique" is actually the correct usage.